### ISSUE TITLE
Wrong facing tile in StructureConstruct.xml

### ISSUE THREAD

**Issue by Coelancanth** [1dot13/gamedir#81]
Should be 

<STRUCTURE>
		<usCreationItem>1542</usCreationItem>
		<usItemStatusLoss>1</usItemStatusLoss>
		<szTileSetDisplayName>Concertina wire</szTileSetDisplayName>
		<szTileSetName>spot_1.sti</szTileSetName>
		<dCreationCost>1.0</dCreationCost>
		<fFortifyAdjacentAdjustment>1</fFortifyAdjacentAdjustment>
		<northfacingtile>2</northfacingtile>
		<northfacingtile>4</northfacingtile>
		<southfacingtile>2</southfacingtile>
		<southfacingtile>4</southfacingtile>
		<eastfacingtile>3</eastfacingtile>
		<eastfacingtile>5</eastfacingtile>
		<westfacingtile>3</westfacingtile>
		<westfacingtile>5</westfacingtile>

instead of
<STRUCTURE>
		<usCreationItem>1542</usCreationItem>
		<usItemStatusLoss>1</usItemStatusLoss>
		<szTileSetDisplayName>Concertina wire</szTileSetDisplayName>
		<szTileSetName>spot_1.sti</szTileSetName>
		<dCreationCost>1.0</dCreationCost>
		<fFortifyAdjacentAdjustment>1</fFortifyAdjacentAdjustment>
		<northfacingtile>3</northfacingtile>
		<northfacingtile>5</northfacingtile>
		<southfacingtile>3</southfacingtile>
		<southfacingtile>5</southfacingtile>
		<eastfacingtile>2</eastfacingtile>
		<eastfacingtile>4</eastfacingtile>
		<westfacingtile>2</westfacingtile>
		<westfacingtile>4</westfacingtile>

tldr: 
should be 2424, 3535. Instead of 3535,2424

**Comment by kmoiwyh** [comment]
> ![co3-5](https://private-user-images.githubusercontent.com/58940527/313402991-715bdcb7-0321-438d-b27a-11cfd135b982.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTE3ODIzNDgsIm5iZiI6MTcxMTc4MjA0OCwicGF0aCI6Ii81ODk0MDUyNy8zMTM0MDI5OTEtNzE1YmRjYjctMDMyMS00MzhkLWIyN2EtMTFjZmQxMzViOTgyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMzAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzMwVDA3MDA0OFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTlmZjI4MzEzODRlOWEzYTc1OWU4NWEyOWYwN2Y0MWEwNDc3ZGQ4MzhjYmM2NzI0NTQxMmEzMjg4ZGQ0NWZiZGYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.I8KMLd7wcWzS4-p3ZqI2gl_LPWLdqFYQgztuGPBLHbU) concertina in spot_1.sti looks like that
> 
> I'm not realy sure ...

It is incorrect. Wrong direction feedback in the game.
![bad1](https://github.com/1dot13/gamedir/assets/117851345/7b438140-a0c8-4561-b700-016fbcb9434b)

**Comment by kmoiwyh** [comment]
> Concertina wire

Please use the defensive fortification function

Uploading Wrong direction of Concertina wire.mp4…

**Comment by kmoiwyh** [comment]
<fFortifyAdjacentAdjustment>0</fFortifyAdjacentAdjustment> Alright, there's no problem now. Solved.
